lib/consolidate.js

```diff
@@ -1034,6 +1034,9 @@ exports.nunjucks.render = function(str, options, fn) {
   return promisify(fn, function (fn) {
     try {
       var engine = requires.nunjucks || (requires.nunjucks = require('nunjucks'));
+      if (options.views) engine.configure(options.views);
+      if (options.settings && options.settings.views) engine.configure(options.settings.views);
```

Comment:
Is this really a way people would pass in the configuration? Or would it be fine to only use the views as configuration?